RELEASE_NOTES.md

```diff
@@ -14,4 +14,4 @@
 
 ## Bug Fixes
 
-<!-- Here goes notable bug fixes that are worth a special mention or explanation -->
+- Removed unused imports
```

Comment:
Is that really a _notable_ bug fix?

Reply:
Oh, and then it was removed again

Reply:
Not sure what you mean. No release note is being removed in any commit here.

Reply:
If you mean whether `Removed unused imports` is notable or not, then yes. Otherwise the gRPC gateway build fails.

Reply:
Good point. Done: https://github.com/frequenz-floss/frequenz-api-microgrid/issues/228